server/index.ts

```diff
@@ -80,60 +82,92 @@ export const configSchema = schema.object({
       buttonstyle: schema.string({ defaultValue: '' }),
     }),
   }),
-  multitenancy: schema.maybe(schema.object({
-    enabled: schema.boolean({ defaultValue: false }),
-    show_roles: schema.boolean({ defaultValue: false }),
-    enable_filter: schema.boolean({ defaultValue: false }),
-    debug: schema.boolean({ defaultValue: false }),
-    tenants: schema.object({
-      enable_private: schema.boolean({ defaultValue: true }),
-      enable_global: schema.boolean({ defaultValue: true }),
-      preferred: schema.arrayOf(schema.string(), { defaultValue: [] }),
+  multitenancy: schema.maybe(
```

Comment:
not an issue, but is it possible to add unit or integration test for these schema validation so if someone accidentally revise this, we could identify impact easily.

Reply:
yes, we definitely should have tests for our code.

my plan is to add tests later, when adding other auth types, it is very likely to refactor a big portion of the code, so will add tests after the code refactor is complete